### PR TITLE
Orders api-docs review

### LINKED ISSUE #63: Add endpoint to Orders Exports
Issue https://github.com/DobaTech/thanos/issues/689 will add an endpoint (`POST /orders/search/export/` and it will change how `POST /orders/export/` works. See https://github.com/DobaTech/thanos/issues/689#issuecomment-355439216. Once 689 is done the API docs will need to be updated.

### LINKED ISSUE #85: order allocation response change
In the call /orders/allocation/UUID

The response has changed to instead of returning a 204 NO Content to now be a 200 OK.  

The structure of the response is now:
```json
[
  {
    "f69e76cb-63b3-4f3a-b2fe-f2c45225f53d": "Allocated"
  },
  {
    "021193f9-97b0-4a05-9081-a1b0e6dd82b1": "Allocated"
  },
  {
    "70b4d1fe-a7b0-44e9-a376-627af29c860e": "Item Not Found"
  }
]

```
This is the line_item_uuid and the status of the line item, so that it's explicit what was changed.

### LINKED ISSUE #106: order status conjunction is incorrect
As noted in https://github.com/CruxConnect/crux-ui/issues/572 the backend for the only conjunction is not implemented. (https://github.com/CruxConnect/crux-api/issues/944). We need to remove the documentation for it and open a task relying upon https://github.com/CruxConnect/crux-api/issues/944 to restore it.

- [ ] Remove only from order search conjunction
- [ ] Add issue reliant upon https://github.com/CruxConnect/crux-api/issues/944 to restore the documentation. Assign to the awaiting outside effort milestone and add the holding pattern label.

### LINKED ISSUE #107: Orders API Review
Review existing the Order API docs. 

- [x] Get Orders POST '/orders/'
- [x] Get Orders Export POST '/orders/export/'
- [x] Get Order Detail  `/orders/${uuid}/`
- [x] Get Order Fees - Retailer POST '/orders/fees/'
- [x] Create Order - Retailer
- [x] Cancel Order
- [x] Cancel Order Item - Retailer

Add following docs
- [x] Add Orders Tracking Post `/orders/tracking/${orderUUID}
- [x] Allocate Order Put `/orders/allocation/${orderUUID}/`

Incorporate already identified issues.  To include:
- [x] #63 Add endpoint to Orders Export
- [x] #85 Order allocation response change
- [x] #106 Order status conjunction

### PR DESCRIPTION
**Pull request by nickcoleman**
General review and update of the Orders API documentation.

Fixes #63  Add endpoint to Orders Export
Fixes #85  Order allocation response change
Fixes #106 Order status conjunction
Fixes #107 Orders Api Review

Paw updated to remove environment authentication data

### CODE REVIEW

File: _ordering/allocate_order_supplier.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+#### Order Item Object
+
+line_item_uuid
+: (string) The Order Item ID is the sku_id for the product purchased by the Retailer
+
+allocation
+: (object) The Allocation Object parameter includes the quantites accepted, rejected, and backordered.
+
+#### Allocation Object
```
Comment (Gidgidonihah) [inline _ordering/allocate_order_supplier.md]: Didn't you create an include for this?

File: _ordering/create_order_retailer.md [hunk 2]
```diff
@@ -30,31 +27,31 @@ right_code: |
...
     "retailer": {
-      "name": "projectthanos",
-      "uuid": "e7409ece-e923-4aa8-a41b-4aacb9e475be",
+      "name": "Crux Retailer",
+      "uuid": "93204006-fcdc-458c-8f81-13a7337992ae",
       "user": {
-        "name": "Crux  User",
-        "email": "user@mycompany.com"
+        "name": "Roy Breslawski",
+        "email": "rbreslawski@cruxretailer.com"
```
Comment (Gidgidonihah) [inline _ordering/create_order_retailer.md]: Oops.

File: _ordering/get_order_detail.md [hunk 3]
```diff
@@ -1,119 +1,64 @@
...
     "retailer": {
-      "name": "projectthanos",
-      "uuid": "e7409ece-e923-4aa8-a41b-4aacb9e475be",
+      "name": "Crux Retailer",
+      "uuid": "93204006-fcdc-458c-8f81-13a7337992ae",
       "user": {
-        "name": "Crux User",
-        "email": "user@mycompany.com"
+        "name": "Roy Breslawski",
+        "email": "rbreslawski@cruxretailer.com"
```
Comment (Gidgidonihah) [inline _ordering/get_order_detail.md]: Oops.

File: _ordering/get_order_detail.md [hunk 4]
```diff
@@ -1,119 +1,64 @@
...
     "address": {
-      "name": "James Smith",
-      "business_name": "Price-Lloyd",
-      "address1": "1878 Randy Valleys\nAnthonybury, OH 51137",
-      "address2": "127 Cox Spurs\nLake Deborah, PW 48750",
-      "city": "Lake Coltonstad",
-      "state": "Ohio",
-      "postal_code": "90181"
+      "name": "Bob Iger",
+      "business_name": "NBC",
```
Comment (Gidgidonihah) [inline _ordering/get_order_detail.md]: Bob Iger is decidedly not in charge of NBC. That would be Disney. Cmon, man.

File: _ordering/get_order_fees_retailer.md [hunk 5]
```diff
@@ -25,19 +20,20 @@ right_code: |
...
   {: title="Response" }
 
 ---
 Before placing an order, use this call to return the total Fees for the Order. This may be used a number of ways. Perhaps you would like to make sure you are charging enough for an order and would like a "sanity check" on the price before committing to the potential buyer. This API call allows you to get the full, all-inclusive price for the Order.
 
+
 ### Request Parameters:
 
-#### Required:
+##### Required:
```
Comment (Gidgidonihah) [inline _ordering/get_order_fees_retailer.md]: Why the increase in level?

File: _ordering/get_orders_export.md [hunk 6]
```diff
@@ -1,61 +1,107 @@
 ---
-title: /orders/export/
-name: Get Orders Export
-position: 3.01
+title: /orders/fees/
```
Comment (Gidgidonihah) [inline _ordering/get_orders_export.md]: Um... This seems wrong for the filename.

File: _ordering/allocate_order_supplier.md (hunk 1 above)
Comment (nickcoleman) [reply _ordering/allocate_order_supplier.md]: good catch

File: _ordering/create_order_retailer.md (hunk 2 above)
Comment (nickcoleman) [reply _ordering/create_order_retailer.md]: good catch

File: _ordering/get_order_detail.md (hunk 3 above)
Comment (nickcoleman) [reply _ordering/get_order_detail.md]: good catch

File: _ordering/get_order_detail.md (hunk 4 above)
Comment (nickcoleman) [reply _ordering/get_order_detail.md]: But, Obviously caused by an insidious plot by Bog Iger to corrupt our backend test data generation and further his cause to rule the world through subliminal messages.  Very Mickey Mouse.

File: _ordering/get_order_fees_retailer.md (hunk 5 above)
Comment (nickcoleman) [reply _ordering/get_order_fees_retailer.md]: So it matched the "Optional" level.  I can decrease them both if that looks better

File: _ordering/get_orders_export.md (hunk 6 above)
Comment (nickcoleman) [reply _ordering/get_orders_export.md]: Hmm ... fixed w/ a fresh import

File: _ordering/get_order_fees_retailer.md (hunk 5 above)
Comment (Gidgidonihah) [reply _ordering/get_order_fees_retailer.md]: Optional is only 4. You increased required to 5.

Comment (nickcoleman) [reply _ordering/get_order_fees_retailer.md]: you are right and I am blind 🙈

**Pull request merged**